pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -29,7 +29,7 @@
 
 
 # Initialize MongoDB connection
-initialize()
+# initialize()
```

Comment:
@dkliban @dralley , why do we need to initialize a mongodb connection from the functional tests code? How do we use it? I thought that Pulp which runs in a container does it when we trigger a task to migrate from pulp 2 to pulp 3.

Reply:
Found it. We use mongo to find the list of repositories in pulp2.